c7n/policy.py

```diff
@@ -586,13 +586,22 @@ def provision(self):
 class PeriodicMode(LambdaMode, PullMode):
     """A policy that runs in pull mode within lambda.
 
-    Runs Custodian in AWS lambda at user defined cron interval.
+    Runs Custodian in AWS lambda at user defined cron interval using either
+    EventBridge Scheduler (if scheduler_role provided) or EventBridge Rules.
     """
 
     POLICY_METRICS = ('ResourceCount', 'ResourceTime', 'ActionTime')
 
     schema = utils.type_schema(
-        'periodic', schedule={'type': 'string'}, rinherit=LambdaMode.schema)
+        'periodic',
+        schedule={'type': 'string'},
+        timezone={'type': 'string'},
+        start_date={'type': 'string'},
```

Comment:
we've got a mix of underscore and hyphens on lambda modes, I think preference for a new key would be to use hyphens.

Reply:
changed all the properties to use dashes instead of underscores